xclim/testing/tests/test_indices.py

```diff
@@ -1546,52 +1546,6 @@ def test_indice_against_icclim(self, cmip3_day_tas):
         np.testing.assert_array_equal(icclim, ind)
 
 
-@pytest.mark.skip("Fire season computation is not the same as GFWED")
```

Comment:
Why is this gone ?

Reply:
Nothing to do with this PR, sorry. I just realized it was there when working on it. These tests are now in `test_fwi.py`, since #704.